### DISCUSSION TITLE
Add SSH:22 (or Mongodb:27117) ports/config to container

### DISCUSSION THREAD

**Discussion by cwmoriarty** [hassio-addons/addon-unifi#437] [Ideas]
# Problem/Motivation

I'd like to SSH into the controller to make changes to mongodb when the controller has decided to block a WAP, for example. 

If the controller needs administration in any way (it often does), there's no way to obtain shell access.

**Comment by frenck** [comment]
I have no plan to add such a thing to the container.

../Frenck

**Reply by cwmoriarty** [reply]
Is there a good reason not to add it as an option? This is a very real and somewhat common situation.

**Reply by frenck** [reply]
If you need to access it this way, it makes more sense to not use the addon.

../Frenck

**Reply by cwmoriarty** [reply]
I don't understand the rationale there. Because it's containerized? Because it's within HACS?

**Reply by frenck** [reply]
The rational is: I am not adding it, because it is jot needed. If you do need it, this addon is not for you.

**Reply by frenck** [reply]
I won't accept such a PR, as per above.
